unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/Passport/Tests/PassportHUDControllerShould.cs

```diff
@@ -0,0 +1,81 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using NSubstitute;
+using NUnit.Framework;
+using DCL.Social.Passports;
+using DCL;
+using SocialFeaturesAnalytics;
+
+public class PassportHUDControllerShould
+{
+    private PlayerPassportHUDController controller;
+    private PlayerPassportHUDView view;
+    private PassportPlayerInfoComponentController playerInfoController;
+    private PassportPlayerPreviewComponentController playerPreviewController;
+    private PassportNavigationComponentController passportNavigationController;
+    private StringVariable currentPlayerInfoCardId;
+    private IUserProfileBridge userProfileBridge;
+    private ISocialAnalytics socialAnalytics;
+    private DataStore dataStore;
+    private IProfanityFilter profanityFilter;
+    private IFriendsController friendsController;
+
+    [SetUp]
+    public void SetUp()
```

Comment:
is missing many test cases of the class, are you planning to do it later?

Reply:
yes :)